packages/@ama-sdk/core/src/plugins/url-rewrite/url-rewrite.spec.ts

```diff
@@ -12,6 +12,7 @@ describe('URL Rewrite Request Plugin', () => {
 
   beforeEach(() => {
     options = {
+      method: 'get',
```

Comment:
was this mandatory, or you added it to have the tests more explicit ?

Reply:
It is mandatory. I turned it to mandatory to avoid defaulting to different method according to the request shooter mechanism.
No impact other than test as we always explicit the method

Reply:
I see. The `method` is always passed in the request options in the `prepareOptions` method of the clients we have.